tslint-server/src/server.ts

```diff
@@ -361,8 +366,9 @@ function doValidate(conn: server.IConnection, document: server.TextDocument): se
 
 	let result: tslint.LintResult;
 	try { // protect against tslint crashes
-		let tslint = new linter(fsPath, contents, options);
-		result = tslint.lint();
+		let tslint = new linter(options);
```

Comment:
Do you want to pass in the configuration from the `Linter.createConfiguration` method here? Down below in the `tslint.lint(..)` would then come `tslint.lint(fsPath);`